platforms/core-runtime/client-services/src/main/java/org/gradle/launcher/daemon/client/DefaultDaemonConnector.java

```diff
@@ -50,8 +50,8 @@
 import java.util.List;
 
 import static java.lang.Thread.sleep;
-import static org.gradle.launcher.daemon.server.api.DaemonStateControl.State.Canceled;
```

Comment:
Split `State` out of `DaemonStateControl` so they can move to different projects.